queries/python.scm

```diff
@@ -27,6 +27,58 @@
   (with_statement)
 ] @statement
 
+;;!! a = 25
+;;!      ^^
+;;!   xxxxx
+;;!  ------
+(assignment
+  (_) @_.leading.start.endOf
+  .
+  right: (_) @value @_.leading.end.startOf
+) @_.domain
```

Comment:
fwiw I do think this pattern and the following one (`augmented_assignment`) are potentially a compelling use case for `#has-type?` predicate, as it would avoid duplication. @AndreasArvidsson wdyt?

Reply:
To me it is the same as dictionary vs dictionary_comprehension as discussed in https://github.com/cursorless-dev/cursorless/commit/36c2f4c7e5227086cf9ed098a5c8d787fea063d5

Reply:
Agreed it is quite similar. In that case we managed to avoid duplication another way, but I don't see a way out of it here other than `has-type`